charter-template.html

```diff
@@ -254,8 +254,8 @@ <h3>Timeline</h3>
 	  <h2>Success Criteria</h2>
 
     <!-- Testing and interop -->
-	  <p><span class='todo'>Remove this clause if the Group does not intend to move to REC:</span> In order to advance to
-		  <a href="https://www.w3.org/policies/process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have
+	  <p><span class='todo'>Remove this clause if the Group does not intend to move to REC:</span> In order to advance beyond
+		  <a href="https://www.w3.org/policies/process/#RecsCR" title="Candidate Recommendation">Candidate Recommendation</a>, each normative specification is expected to have
```

Comment:
Your PR didn't change this, but it seems unnecessary to have a `title` on the link that just repeats the content of the link.

```suggestion
		  <a href="https://www.w3.org/policies/process/#RecsCR">Candidate Recommendation</a>, each normative specification is expected to have
```